src/attr/__init__.pyi

```diff
@@ -64,9 +65,15 @@ NOTHING: object
 def Factory(factory: Callable[[], _T]) -> _T: ...
 @overload
 def Factory(
```

Comment:
Since Literal doesn't exist in older pythons (and will get turned to Any) you'll probaby want to use
```
if sys.version_info >= (3, 8):
```
to keep the old overload.

Also if you use 
```
import typing
if sys.version_info >= (3, 8):
    typing.Literal
```

You avoid the error message on import.